src/openassetio-core/src/hostApi/Manager.cpp

```diff
@@ -11,6 +11,30 @@
 
 namespace openassetio {
 inline namespace OPENASSETIO_CORE_ABI_VERSION {
+namespace {
+// Takes a BatchElementError and throws and equivalent exception.
+void throwFromBatchElementError(std::size_t index, const BatchElementError &error) {
```

Comment:
Wouldn't that necessarily make this method public? We wanted to avoid that, as it's not functionality we really want to provide users or put into the public interface.

Reply:
Good point. Lets just remember we have this and see if we end up repeating it anywhere.

Reply:
Whatever you folks prefer, all good points, was just an ill-considered consider.